src/main/java/mate/academy/spring/dao/impl/MovieSessionDaoImpl.java

```diff
@@ -57,4 +58,51 @@ public Optional<MovieSession> get(Long id) {
             throw new DataProcessingException("Can't get a movie session by id: " + id, e);
         }
     }
+
+    @Override
+    public MovieSession update(MovieSession movieSession) {
+        Session session = null;
+        Transaction transaction = null;
+        try {
+            session = sessionFactory.openSession();
+            transaction = session.beginTransaction();
+            session.update(movieSession);
+            transaction.commit();
+            return movieSession;
+        } catch (Exception e) {
+            if (transaction != null) {
+                transaction.rollback();
+            }
+            throw new DataProcessingException("Can`t update movie session with id "
+                    + movieSession.getId(), e);
```

Comment:
we can provide more info in the error message
```suggestion
            throw new DataProcessingException("Can`t update movie session: "
                    + movieSession, e);
```